standards/peer-review.md

```diff
@@ -1,41 +1,38 @@
-  - Reviewers MAY come from any team within NYPL Digital
-  - Teams MAY create [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) files to establish a default group of reviewers
-  - Experience SHOULD NOT be taken into account when requesting review
-  - Requests for specific feedback MAY be requested by tagging specific engineers in PR comments
+These guidelines establish the principles and processes for Pull Request (PR) reviews. Following them ensures we maintain high code quality, share knowledge effectively, and foster a collaborative and supportive engineering culture.
 
-- Engineers marked as reviewers SHOULD complete the review in a timely manner
-  - This period SHOULD be agreed upon within individual development teams, 24-48 hours has proven to be a reasonable starting place
-  - Code review SHOULD take some time (more than a quick LGTM) but should not take more than 1-2 hours
-  - If a code review is very lengthy or complex, reviewers MAY request that the PR be broken up into smaller pieces and/or request a walkthrough/live review session
- 
-- If changes are requested (or made in response to questions) the requesting engineer SHOULD re-request review when ready
-  - This helps reviewers track the current status of open PRs and plan time for review work
-  - Additional rounds of review SHOULD be completed until the PR can be either merged or closed
-  - To help track PR status teams MAY use GitHub labels (e.g. `Changes Requested`, `Needs Review` and `Ship It`) to help socialize status.
+All pull requests (PRs) **must** be reviewed by at least one other engineer before being merged. Exemptions to this rule require a specific, written policy agreed upon by the team and approved by an Engineering Manager or Director.
 
-- Engineers SHOULD utilize the Draft PR status to socialize work in progress and to solicit feedback
-  - Reviewers SHOULD NOT use the "request changes" option on draft PR reviews
-  - When a draft PR is ready for full review the requesting engineer MUST update its status and request (or re-request) review from relevant team members
+- **PR reviews are a core responsibility**
+  This isn't an "extra" task; it's a fundamental part of every engineer's job, regardless of level. Actively participating in reviews is crucial for code quality, knowledge sharing, and team success.
 
-# Peer Review
+- **Be timely & responsive**
+  Prioritize reviews to keep the development flow moving. Aim for initial feedback within a **24-48 hour** timeframe. If you have thoughts but can't give a full review, consider setting up a 30-minute walk-through with the author. After you have pushed updates based on feedback, **re-request a review** from the original reviewers to notify them that the PR is ready for another look. This helps everyone track the status and keeps the process moving.
```

Comment:
Sometimes, the "re-request review" gets lost and many devs miss the notification. Can we add something to the effect of "Or message your team on slack that the PR has been updated with feedback. Do what works on your team!"